holidays/index.js

```diff
@@ -0,0 +1,19 @@
+export {default as at} from './at.js';
```

Comment:
I would prefer to have the holidays as simple JSON files for interoperability reasons. Seems like [rollup-plugin-json](https://github.com/rollup/rollup-plugin-json) could come in handy for this. Does this make sense and can you try it?